Backpack/NavigationBar/Classes/BPKNavigationBar.m

```diff
@@ -259,6 +261,25 @@ - (UIView *)borderView {
     return _borderView;
 }
 
+- (BPKNavigationBarButton *)leftButton {
+    if (!_leftButton) {
+        _leftButton = [[BPKNavigationBarButton alloc] init];
+        _leftButton.translatesAutoresizingMaskIntoConstraints = NO;
+        _leftButton.imagePosition = BPKButtonImagePositionLeading;
+        _leftButton.hidden = YES;
```

Comment:
Should this have a chevron too?

Reply:
We don't have currently the appropriate asset with the correct inner padding inside the `BPKIcon` source. Although in the app there's an asset called `backArrow` which could be injected inside the `BPKNavigationButton`, We need to include this eventually in the `BPKIcon` too.

Reply:
Cool. Will leave it as is for now then.
Do you mind if I push a change so that we have an example with buttons and another without?

Reply:
That would be great thanks!